recipes-kernel/linux/4.19/defconfigs/xenclient-stubdomain/defconfig

```diff
@@ -138,10 +139,11 @@ CONFIG_ANON_INODES=y
 CONFIG_SYSCTL_EXCEPTION_TRACE=y
 CONFIG_HAVE_PCSPKR_PLATFORM=y
 CONFIG_BPF=y
-# CONFIG_EXPERT is not set
+CONFIG_EXPERT=y
```

Comment:
Why is this needed? Just curious.

Reply:
VT & VGA_CONSOLE are hidden unless we enable EXPERT.  With EXPERT enabled, we can disable them.